public/_posts/_articles/marco-worms/yearn-governance/es.md

```diff
@@ -0,0 +1,128 @@
+---
+title:  "Explicación de la gobernanza de Yearn: propuestas, token $YFI y ejecución"
+image:
+  src: ./image1.jpg
+  width: 900
+  height: 482
+date: '2022-04-19'
+author: Marco_Worms
+translator: RestlessMik3
+---
+
+![](./image1.jpg?w=900&h=482)
+
+Cualquiera puede discutir o proponer ideas en Yearn's [Discord](https://discord.com/invite/6PNv2nF), [Twitter](https://twitter.com/iearnfinance), [Reddit](https://www.reddit.com/r/yearn_finance), [Pool of Ideas](https://yearnfinance.notion.site/yearnfinance/Pool-of-Ideas-d75383ade9154d8bb6163388c6c2b39b), [Telegram](https://t.me/yearnfinance/), y [Foro de Gobernanza](https://gov.yearn.finance/). A medida que maduran las propuestas, se convierte en un voto estructurado en el Foro de Gobernanza donde el [token YFI](https://www.coingecko.com/en/coins/yearn-finance) se usa como poder de voto.
+
+- Los poseedores de tokens pueden votar en [La página de snapshots de Yearn](https://yearn.snapshot.page/#/), ¡no hay consumo de gas en la transacción de votación!
+- También hay un [repositorio de gobierno retirado](https://docs.yearn.finance/contributing/governance/proposal-repository) que ahora no se usa pero se puede explorar para aprender sobre la historia de Yearn.
+
+El tipo más común de propuestas en el foro son las Propuestas de mejora de Yearn (YIP), pero hay otros tipos cubiertos en la [implementación actual de la gobernanza de Yearn (YIP-61)](https://gov.yearn.finance/t/yip-61-gobernanza-2-0/10460):
+
+- **Propuesta de mejora anual (YIP):** Una propuesta para ejecutar cualquier poder delegado a los titulares de YFI o fuera del alcance de los poderes enumerados
+- **Propuesta de delegación de año (YDP):** Una propuesta para cambiar la forma en que un yTeam tiene poder sobre un alcance
```

Comment:
AS-IS: **Propuesta de delegación de año (YDP):** Una propuesta para cambiar la forma en que un yTeam tiene poder sobre un alcance

TO-BE: **Propuesta de delegación de año (YDP):** Una propuesta para cambiar la forma en que un yTeam tiene poder sobre un área

Reply:
Done